mellotrainer/vehicles.lua

```diff
@@ -75,7 +77,14 @@ end)
 
 RegisterNUICallback("vehspawn", function(data, cb)
 	local playerPed = GetPlayerPed(-1)
-	local x, y, z = table.unpack(GetOffsetFromEntityInWorldCoords(playerPed, 0.0, 7.5, 0.0))
+	local x, y, z
+
+	if ( featureSpawnInsideCar ) then 
```

Comment:
Think this should be changed to be a spawn setting and not default functionality. All vehicles should spawn the same regardless of `featureSpawnInsideCar`. There should be a `featureSpawnCarAtPlayerPosition` or something similar to determine this.